instructions for setting up new repository from template (#75)
codeforamerica/form-flow-starter-app: README.md

# Using this as a template repository

> 1. Navigate to [this link](https://github.com/codeforamerica/form-flow-starter-app/generate) to create a new repository from the `form-flow-starter-app`.

Reviewer: Instead of using `this link` for the link text, incorporate into the nature structure of the sentence.

Here's a [resource on accessibility for link text](https://usability.yale.edu/web-accessibility/articles/links#link-text).

```suggestion
1. Create a [new repository from the `form-flow-starter-app` template](https://github.com/codeforamerica/form-flow-starter-app/generate).
```
